pkg/oauth/apis/oauth/v1/types.go

```diff
@@ -117,6 +120,10 @@ type OAuthClient struct {
 	// AccessTokenMaxAgeSeconds overrides the default access token max age for tokens granted to this client.
 	// 0 means no expiration.
 	AccessTokenMaxAgeSeconds *int32 `json:"accessTokenMaxAgeSeconds,omitempty" protobuf:"varint,8,opt,name=accessTokenMaxAgeSeconds"`
+
+	// AccessTokenTimeoutSeconds enables the Token Timeout feature.
```

Comment:
I believe this comment refers to the TokenConfig struct's field of the same name.

Reply:
No, it is just an obsolete comment, it used to be that you had to enable the feature by setting the oauthclient parameter.